deviceaccess.py

```diff
@@ -15,15 +15,16 @@
 
 from __future__ import annotations
 
-from typing import Sequence
+from typing import Sequence, Union
 import _da_python_bindings as pb
 import numpy as np
 from _da_python_bindings import AccessMode, DataValidity, TransferElementID, VersionNumber
 from abc import ABC
-
+from functools import reduce
```

Comment:
Python question: If I import the deviceaccess module, will I get `reduce` imported from functools as well automatically?

If yes: We should avoid all `from X import Y` or `import X as Y` constructions, as they will spoil the top-level namespace of the using Python scripts.

Reply:
We do want to have `AccessMode` etc. available in the imported namespace, though.
I am not sure about Sequence and Union, af this will get a bit wordy for the type hints.

I think it is pretty a pretty "normal" behaviour to have a "poluted namespace", but I agree to keep it low.